Services.Test/DeviceModelsTest.cs

```diff
@@ -196,6 +154,13 @@ public void ItInvokeCustomDeviceModelsServiceWhenDeleteDeviceModel()
         [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
         public void ItThrowsExceptionWhenDeleteDeviceModelFailed()
         {
+            /* SomeException is required to verify that the exception thrown is the one 
+             * configured inn Arrange expression, and the test doesn't pass for the wrong 
```

Comment:
typo "inn".  This comment is very useful, could you add a similar one to `SomeException.cs` ?

Reply:
copied the comment